### ISSUE TITLE
OpenChain Self Certification Questionnaire - Bullet Arrow color

### ISSUE THREAD

**Issue by shanecoughlan** [OpenChain-Project/Online-Self-Certification-Web-App#70]
<img width="137" alt="screen shot 2019-01-16 at 16 38 47" src="https://user-images.githubusercontent.com/3616071/51233658-c9f7af00-19ad-11e9-92e8-06d7707d3f2f.png">

The bullet arrow should be white to match the rest of the UI here and to improve accessibility.

**Comment by goneall** [comment]
Fixed on the I18N branch
